test/e2e/lib/components/notices-component.js

```diff
@@ -15,18 +15,52 @@ import AsyncBaseContainer from '../async-base-container';
 
 export default class NoticesComponent extends AsyncBaseContainer {
 	constructor( driver ) {
-		super( driver, By.css( '#notices' ), null, config.get( 'explicitWaitMS' ) * 3 );
+		super( driver, By.css( '.wpcom-site' ), null, config.get( 'explicitWaitMS' ) * 3 );
 	}
 
-	async inviteMessageTitle() {
-		const selector = By.css( '.invites__title' );
+	async isSuccessNoticeDisplayed( click = false ) {
+		const selector = By.css( '.notice.is-success' );
+		const actionSelector = By.css( '.notice.is-success a' );
+		const isDisplayed = await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
+		if ( click === true ) {
+			await driverHelper.clickWhenClickable( this.driver, actionSelector );
+		}
+		return isDisplayed;
+	}
+
+	async isErrorNoticeDisplayed() {
+		const selector = By.css( '.notice.is-error' );
+		return await driverHelper.isEventuallyPresentAndDisplayed( this.driver, selector );
+	}
+
+	async getNoticeContent() {
+		const selector = By.css( '.notice .notice__text' );
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
 		return await this.driver.findElement( selector ).getText();
 	}
 
-	async followMessageTitle() {
-		const selector = By.css( '#notices .notice__text' );
+	async dismissNotice() {
+		const selector = By.css( '.notice.is-dismissable .notice__dismiss' );
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
+		return await driverHelper.clickWhenClickable( this.driver, selector );
+	}
+
+	async getInviteNoticeContent() {
```

Comment:
Completely minor, so feel free to ignore :) I just not really sure if we really need to mix reusable component with page specific methods. Not sure what's a better place for this method is, but having it here seems a bit strange to me.

Reply:
This was here before and I didn't change it at all. Now I've changed it to use [`getNoticeContent`](https://github.com/Automattic/wp-calypso/pull/33194/commits/c6b58c0e242d68558583eb72751764af1f156e5d) and it works well 👌

Reply:
Noice!